setup.py

```diff
@@ -45,7 +45,7 @@
         'typing_extensions',    # 0.20 MB
         'python-json-logger',   # 0.05 MB
         'iso8601',              # 0.07 MB
-        'click',                # 0.60 MB
+        'click>=8.0.0',         # 0.60 MB
```

Comment:
Is the solution not compatible with 7.x.x?

Reply:
have not tried it yet, lemme check